source/docs/software/wpilib-tools/smartdashboard/verifying-smartdashboard-is-working.rst

```diff
@@ -0,0 +1,71 @@
+Verifying SmartDashboard is working
+===================================
+
+Window title connection indicator
+---------------------------------
+
+SmartDashboard will automatically include the connection status and IP address of the NetworkTables source in the title of the window.
+
+.. image:: images/verifying-smartdashboard-is-working/connection-indicator-disconnected.png
+  :width: 350
+
+.. image:: images/verifying-smartdashboard-is-working/connection-indicator-connected.png
+  :width: 350
+
+Connection indicator widget
+---------------------------
+
+SmartDashboard includes a connection indicator widget which will turn red or green depending on the connection to NetworkTables, usually provided by the roboRIO. For instructions to add this widget, look at :ref:`Adding a Connection Indicator <docs/software/wpilib-tools/smartdashboard/smartdashboard-intro:Adding a Connection Indicator>` in the SmartDashboard Intro.
+
+Minimal robot program
+---------------------
+
+.. tabs::
+
+  .. code-tab:: java
+
+      public class Robot extends TimedRobot {
+        double counter = 0.0;
+
+        public void teleopPeriodic() {
+          SmartDashboard.putNumber("Counter", counter++);
+        }
+      }
+
+  .. code-tab:: c++
+
+      #include "Robot.h"
+      float counter = 0.0;
+
+      void Robot::TeleopPeriodic() {
+          frc::SmartDashboard::PutNumber("Counter", counter++);
+      }
+
+This is a minimal robot program that writes a value to the SmartDashboard. It simply increments a counter 50 times per second to verify that the connection is working. However, to minimize bandwidth usage, NetworkTables by default will throttle the updates to 10 times per second.
+
+SmartDashboard output for the sample program
+--------------------------------------------
+
+.. image:: images/verifying-smartdashboard-is-working/smartdashboard-output-sample-program.png
+
+The SmartDashboard display should look like this after about 6 seconds of the robot being enabled in Teleop mode. If it doesn't, then you need to check that the connection is correctly set up.
+
+Verifying the IP address in SmartDashboard
+------------------------------------------
+
+.. image:: images/verifying-smartdashboard-is-working/verifying-ip-address.png
+
+If the display of the value is not appearing, verify that the team number is correctly set as shown in this picture. The preferences dialog can be viewed by selecting ``File``, then ``Preferences``.
+
+Using OutlineViewer to verify that the program is working
+---------------------------------------------------------
```

Comment:
```suggestion
Verifying Program with SmartDashboard
-------------------------------------
```

Reply:
I think the new suggestion is confusing, especially since SmartDashboard is both a tool and an API and is thus ambiguous. This uses the OutlineViewer tool to verify that a program with the SmartDashboard API is working. How about Verifying Program using OutlineViewer?

Reply:
Sounds good. I think `Verifying Program using OutlineViewer` is clear.